sdk/bare/common/sys/Logging-Binary-Format.md

```diff
@@ -0,0 +1,40 @@
+# Logging Binary Format
+
+## Overview
+
+The AMDC can dump the logged variables to the UART serial console using either text-based formatting or the binary
+format. This document describes the binary formatting which appears on the UART interface.
+
+## Dumping Variables
+
+Each logged variable must be dumped one at a time (i.e. if the user has 10 slots, each must be dumped one by one).
```

Comment:
I'd like to see a table here that shows a complete data packet, something like this:

| Byte 0-15            | Byte 16-19 | Byte 20 - 23 | Byte 24 - 31 | ... | Byte `32 + 8*(n-1)` - `32 + 8*n-1` | Byte `32 + 8*n` -  `32 + 8*n + 15` |
|---------------------|----------|-----------|------| ------| ------| ---- |
| Header    | Number of samples| Data Type| sample packet 0 | ... | sample packet _n_ | Footer |

Reply:
Added in: 0e63e285403c5c7d36900d9e1960dc58b305300a